package.json

```
@@ -18,7 +18,7 @@
     "url": "git+https://github.com/Developer-DAO/web3-ui/"
   },
   "scripts": {
-    "build": "yarn clean; rollup -c",
+    "build": "yarn clean; preconstruct build; cp LICENSE package.json README.md ./dist",
     "clean": "rimraf dist",
```

```suggestion
    "build": "preconstruct build",
```

`preconstruct` cleans as part of its internal process. We also don't need to copy anything into `dist`, we can just include `LICENSE` and `README.md` later as part of the published module using the `files` key (`package.json` is always included).